src/pages/About.tsx

```diff
@@ -144,8 +146,15 @@ const ParagraphStyle = styled.div`
   & h2 {
     font-size: 1.5em;
   }
+  & .funding-paragraph {
+    display: flex;
+  }
```

Comment:
is it on purpose?

Reply:
Yes, in order to keep it in the same line.

Reply:
And the margin 0.1 for the p, for the same reason.. align-items center didn't work...

Reply:
after removing it, ask for a review
![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/c02523db-649c-45f5-a7ca-81e22fb92487)